selene/conditions.py

```diff
@@ -250,6 +250,20 @@ def match(self, webelement):
 exact_text = ExactText
 
 
+class StrippedText(ElementCondition):
+    def __init__(self, expected_text):
+        self.expected_text = expected_text
+
+    def match(self, webelement):
+        actual_text = webelement.text
+        if not self.expected_text == str(actual_text).strip():
+            raise ConditionMismatchException(expected=self.expected_text, actual=actual_text)
+        return webelement
```

Comment:
I think that an exception message should have stripped text also because it's might be not understandable

Reply:
> I think that an exception message should have stripped text also because it's might be not understandable

@SergeyPirogov you mean?
```
raise ConditionMismatchException(expected=self.expected_texts.strip(), actual=actual_text)
```